framework/db/oci/Command.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+
+namespace yii\db\oci;
+
+
+class Command extends \yii\db\Command
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function bindPendingParams()
+    {
+        foreach ($this->_pendingParams as $name => $value) {
+            if($value[1] === \PDO::PARAM_STR) {
+                $clonedValue = clone $value[0];
+                $this->pdoStatement->bindParam($name, $clonedValue, $value[1], strlen($clonedValue));
```

Comment:
`strlen()` would error with "Warning: strlen() expects parameter 1 to be string, object given" since `$clonedValue` should be an object.